src/main.ts

```diff
@@ -510,29 +509,33 @@ function makeLongUtils(options: Options, bytes: ReturnType<typeof makeByteUtils>
   const longToString = conditionalOutput(
     "longToString",
     code`
-      function longToString(long: ${Long}) {
-        return long.toString();
+      function longToString(int64: { toString(): string }) {
+        return int64.toString();
```

Comment:
Long, bigint, number, and string all share the toString() method.

Reply:
Nice, I like the approach!